# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/windows/ia2/ia2AccessibleValue.h
# Commit: dcc478201cb7
# Full Hash: dcc478201cb75d078d3a5fd33f306c7b9cfcd527
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2021-08-27 03:42:22
# Regressor Bug: 1727643
# File Overlap Count: 1
# Description:
#   Bug 1727643 - P4: Make use of new common methods in platforms and XPCOM. r=morgan
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123698
# ==============================================================================

diff -r 1a9901750064 -r dcc478201cb7 accessible/windows/ia2/ia2AccessibleValue.h
--- a/accessible/windows/ia2/ia2AccessibleValue.h	Thu Aug 26 19:40:05 2021 +0000
+++ b/accessible/windows/ia2/ia2AccessibleValue.h	Thu Aug 26 19:40:06 2021 +0000
@@ -33,6 +33,7 @@
       /* [retval][out] */ VARIANT* minimumValue);
 
  private:
+  Accessible* Acc();
   AccessibleWrap* LocalAcc();
 };
 